components/uniset/ppucd-test.txt

```diff
@@ -0,0 +1,2000 @@
+# Preparsed UCD generated by ICU preparseucd.py
```

Comment:
Please don't include the full *ppucd.txt* in this PR.  You should craft a test data file that runs all code paths that you need to test.

Reply:
Done. I changed the file to be a different subset of PPUCD that has the full data for a single binary property ("WSpace").